control-station/src/components/index.ts

```diff
@@ -1 +1,4 @@
-export { default as Status } from "./Status/Status";
+export { default as ControlPanel } from "./ControlPanel/ControlPanel";
+export { default as Navbar } from "./Navbar/Navbar";
+export { default as SensorData } from "./SensorBoxes/SensorData";
+export { default as Status } from "./status/Status";
```

Comment:
Issue: component directory name for `Status` needs to be capitalized
```suggestion
export { default as Status } from "./Status/Status";
```